### PR TITLE
updated README

### PR DESCRIPTION
**Pull request by RCopeland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,95 @@
+# WARNING
+
+### Spark is currently in `alpha`. It is subject to changes of all sizes at any time and not suitable for production use. Please reach out to the [Spark team](mailto:sparkdesignsystem@quickenloans.com) for guidance.
+
```
Comment (afebbraro) [inline README.md]: Nice on the warning!

Comment (RCopeland) [reply README.md]: it was there already, I just moved it up and made it much bigger : )

File: README.md [hunk 2]
```diff
@@ -1,31 +1,95 @@
...
+  Core consists of all basic patterns, the adoption of which will make the interface
+  of your application feel like a Quicken Loans, Fintech family of companies
+  app. Patterns in core have been identified as necessary to most if
+  not all applications.
 
-## Monorepo
+* Spark-extras
 
-Spark is managed as a monorepo. All of the Spark source code lives in a single repo, but is released as separate packages using [Lerna](https://github.com/lerna/lerna).
+  Extras patterns are things that are not needed on all applications, but when they
```
Comment (yodasw16) [inline README.md]: > Spark Extras are components that may not be needed by all applications, but are consistent with the style of Spark Core.

File: README.md [hunk 3]
```diff
@@ -1,31 +1,95 @@
...
+
+### Spark Core
+
+* Install the npm package
+
+  `npm install --save-dev @sparkdesignsystem/spark-core`
+
+* Import the setup file in your sass build
+
+  `@import node_modules/@sparkdesignsystem/spark';`
```
Comment (yodasw16) [inline README.md]: extra apostrophe at the end of this. Shouldn't there be quotes? like: `@import "node_modules/.../"`. Same with the other import below.

File: README.md [hunk 4]
```diff
@@ -1,31 +1,95 @@
...
+
+  This will bring all spark-core patterns into your build. The spark classes
```
Comment (yodasw16) [inline README.md]: Capitalize Spark and instead of "and should not affect" I would say "so they don't affect.."

File: README.md [hunk 5]
```diff
@@ -1,31 +1,95 @@
...
+  are namespaced and should not affect any existing css.
+
+### Spark Extras
+
+* Install the npm package
+
+  `npm install --save-dev @sparkdesignsystem/spark-extras`
+
+* Import any of the spark extras patterns that you need
```
Comment (yodasw16) [inline README.md]: spark-extras

File: README.md [hunk 6]
```diff
@@ -1,31 +1,95 @@
...
+
+  `@import node_modules/@sparkdesignsystem/spark-extras/<pattern-name>/<pattern-name>';`
+
+  The available patterns as of the current release are:
```
Comment (yodasw16) [inline README.md]: Maybe easier to just put a link to the package here instead of having to maintain a list.

File: README.md [hunk 7]
```diff
@@ -6,22 +6,23 @@
...
+* Spark Core
 
-  Core consists of all basic patterns, the adoption of which will make the interface
-  of your application feel like a Quicken Loans, Fintech family of companies
-  app. Patterns in core have been identified as necessary to most if
-  not all applications.
+  Spark Core contains the basic patterns and components that make up
+  the visual and interaction design of the Quicken Loans family of
+  Fintech products. If your application is expected to look and behave
+  like a Quicken Loans application, this package the starting point.
```
Comment (yodasw16) [inline README.md]: dammit. "_is_ the starting point"

File: README.md [hunk 8]
```diff
@@ -37,25 +38,20 @@ for browser support.**
 
 * Import the setup file in your sass build
 
-  `@import node_modules/@sparkdesignsystem/spark';`
+  `@import "node_modules/@sparkdesignsystem/spark";`
 
-  This will bring all spark-core patterns into your build. The spark classes
-  are namespaced and should not affect any existing css.
+  This will bring all Spark-core patterns into your build. The spark classes
```
Comment (yodasw16) [inline README.md]: this was fine lowercase. If the hyphen is there, lowercase is good.

File: README.md [hunk 9]
```diff
@@ -37,25 +38,20 @@ for browser support.**
...
+  are namespaced do they don't affect any existing css.
```
Comment (yodasw16) [inline README.md]: _so_ not do

**Pull request merged**